### PR TITLE
move all public production config into yaml

### PR DESCRIPTION
**Pull request by chris48s**
- Refs https://github.com/badges/shields-ops/issues/16
- Refs https://github.com/badges/shields-ops/pull/22

### CODE REVIEW

File: config/shields-io-production.yml [hunk 1]
```diff
@@ -6,13 +6,20 @@ public:
...
   ssl:
-    isSecure: true
+    isSecure: false
 
   cors:
     allowedOrigin: ['http://shields.io', 'https://shields.io']
 
+  services:
+    gitlab:
+      authorizedOrigins: 'https://gitlab.com'
```
Comment (calebcartwright) [inline config/shields-io-production.yml:20]: I'm trying to remember why I needed to set this as an env var in the first place when we introduced support for GitLab API based data acquisition, but it's escaping my memory :shrug:

**Review by calebcartwright — APPROVED** [review]
wfm :+1:

File: config/shields-io-production.yml (hunk 1 above)
Comment (chris48s) [reply config/shields-io-production.yml:20]: I reckon what ended up as yaml config and what ended up as env vars was just us being a bit hit and miss over time tbh.

If you mean why we specifically needed to set this for GitLab, GitLab is the only service where we've got credentials set in production and there is no default `authorizedOrigins`. Everything else that takes an `authorizedOrigins` either has a default on `authorizedOrigins` or we allow credentials to be set for self-hosting purposes but don't do it ourselves.

Comment (calebcartwright) [reply config/shields-io-production.yml:20]: > If you mean why we specifically needed to set this for GitLab,

Sorry I forgot to resolve this thread last night. This was the question I had in mind but remembered that was the case

**Pull request merged**